remote_config/src/ios/remote_config_ios.mm

```diff
@@ -173,6 +173,31 @@ static void GetInfoFromFIRRemoteConfig(FIRRemoteConfig *rc_instance, ConfigInfo
   }
 }
 
+static RemoteConfigError ConvertFIRRemoteConfigUpdateError(NSError *error) {
+  switch(error.code) {
+    case FIRRemoteConfigUpdateErrorStreamError:
+      return kRemoteConfigErrorConfigUpdateStreamError;
+    case FIRRemoteConfigUpdateErrorNotFetched:
+      return kRemoteConfigErrorConfigUpdateNotFetched;
+    case FIRRemoteConfigUpdateErrorMessageInvalid:
+      return kRemoteConfigErrorConfigUpdateMessageInvalid;
+    case FIRRemoteConfigUpdateErrorUnavailable:
+      return kRemoteConfigErrorConfigUpdateUnavailable;
+    default:
+      return kRemoteConfigErrorUnimplemented;
+  }
+}
+
+static ConfigUpdate ConvertConfigUpdateKeys(NSSet<NSString *> *keys) {
+  ConfigUpdate configUpdate;
+
+  for (NSString *key in keys) {
+    configUpdate.updated_keys.push_back(util::NSStringToString(key).c_str());
+  }
+
```

Comment:
nitpick: the blank line before and after the for loop could be removed.

Reply:
Updated